src/scope/models/model-component.ts

```diff
@@ -48,7 +48,6 @@ type State = {
   };
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part